pyprof/prof/linear.py

```diff
@@ -50,7 +50,10 @@ def setXWBMNK(self, args):
             x, w, b = args
             assert (x['type'] == w['type'] == "tensor")
             if (b['type'] == "tensor"):
-                assert (len(b['shape']) == 1)
+                # no longer true as b can be of shape [1, XX]
+                # assert (len(b['shape']) == 1)
+                tmp = 1 # dummy statement for commented line above
+                        # TODO: clean up if..elif..else struct
```

Comment:
Please provide complete code or use the Python keyword `pass`.
Can you also provide a test case.